src/chttpd/src/chttpd_sup.erl

```diff
@@ -75,8 +77,9 @@ settings() ->
     [
         {bind_address, config:get("chttpd", "bind_address")},
         {port, config:get("chttpd", "port")},
-        {backlog, config:get("chttpd", "backlog")},
-        {server_options, config:get("chttpd", "server_options")}
+        {backlog, config:get("chttpd", "backlog", ?DEFAULT_BACKLOG)},
```

Comment:
Why not to convert it into `config:get_integer/3`? Is it because we use `undefined` somewhere?

Reply:
Changed.